nixos/platform/network.nix

```diff
@@ -321,8 +321,8 @@ in
 
         resolvconf.extraOptions = [
           "ndots:1"
-          "timeout:1"
-          "attempts:6"
+          "timeout:5" # in correspondence with rfc8767
```

Comment:
ah, i remember: this isn't quite in correspondence. rfc8767 suggests 2 as a common option, but we can't use 2 because kresd isn't able to stale timeout in 1.8 seconds due to granularity.